backend/requirements.txt

```diff
@@ -1,5 +1,5 @@
 asgiref==3.7.2
-Django==5.0.3
```

Comment:
I wasn't able to install `Django 5.0.3` in python 3.9, so thats why I have removed the version. Apart this, every thing else is working perfectly fine ✌️